packages/docusaurus/bin/docusaurus.js

```diff
@@ -187,14 +188,21 @@ cli
     },
   );
 
+cli
+  .command('clear')
+  .description('Clean website')
```

Comment:
I think this change will improve understanding of the CLI command.

```suggestion
  .description('Remove build artifacts')
```